### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by minwoox**
- Dropwizard2 2.1.6 -> 2.1.8
- Dropwizard metrics 4.2.19 -> 4.2.21
- Futures-extra 4.3.1 -> 4.3.3
- Graphql Kotlin 6.5.3 -> 6.5.6
- gRPC 1.57.2 -> 1.58.0
- gRPC Kotlin 1.3.0 -> 1.4.0
- Guava 32.1.2-jre -> 32.1.3-jre
- Jackson 2.15.2 -> 2.15.3
- Jetty
  - 10.0.15 -> 10.0.17
  - 11.0.15 -> 11.0.17
- Kafka 3.5.1 -> 3.6.0
- Micrometer 1.11.3 -> 1.11.5
- Micrometer Tracing 1.1.4 -> 1.1.6
- Netty 4.1.96.Final -> 4.1.100.Final
- Netty incubator 0.0.21.Final -> 0.0.23.Final
- OkHttp 4.11.0 -> 4.12.0
- Reactor 3.5.8 -> 3.5.11
- RxJava3 3.1.6 -> 3.1.8
- Sangria 4.0.1 -> 4.0.2
- Scala2.13 2.13.11 -> 2.13.12
- Spring Framework 6.0.11 -> 6.0.13
- Spring Boot
  - 2.7.14 -> 2.7.16
  - 3.1.2 -> 3.1.4
- Tomcat
  - 8.5.85 -> 8.5.94
  - 9.0.71 -> 9.0.82
  - 10.1.12 -> 10.1.15
- ZooKeeper 3.8.2 -> 3.9.1
- Build and tests
  - Dagger 2.47 -> 2.48.1
  - dgs 7.3.6 -> 7.6.0
  - Errorprone 2.21.1 -> 2.22.0
  - gax-grpc 2.32.0 -> 2.35.0
  - Joor 0.9.14 -> 0.9.15
  - org.ow2.asm 9.5 -> 9.6
  - Snappy 1.1.10.3 -> 1.1.10.5
  - TestContainers 1.18.3 -> 1.19.1

### CODE REVIEW

File: reactor3/src/test/java/com/linecorp/armeria/common/reactor3/ContextAwareFluxTest.java [hunk 1]
```diff
@@ -370,23 +368,6 @@ void connectableFlux() {
...
-        try (SafeCloseable ignored = ctx.push()) {
-            final ConnectableFlux<String> connectableFlux = Flux.just("foo").publish();
-            flux = addCallbacks(connectableFlux.autoConnect(2, disposable -> {
-                assertThat(ctxExists(ctx)).isTrue();
-                future.complete(disposable);
-            }).publishOn(Schedulers.single()), ctx);
-        }
-        flux.subscribe().dispose();
-        flux.subscribe().dispose();
-        assertThat(future.join().isDisposed()).isTrue();
```
Comment (minwoox) [inline reactor3/src/test/java/com/linecorp/armeria/common/reactor3/ContextAwareFluxTest.java:387]: `isDisposed()` doesn't return `true` anymore after the rework of `FluxPublisher`: https://github.com/reactor/reactor-core/pull/3538/files
The Javadoc says that returning `true` isn't guaranteed so I just removed this test case. 
```java
* Optionally return {@literal true} when the resource or task is disposed.
* <p>
* Implementations are not required to track disposition and as such may never
* return {@literal true} even when disposed.
```

**Review by jrhee17 — APPROVED** [review]
Thanks @minwoox ! 🙇 👍 🙇

**Review by trustin — APPROVED** [review]
:bow:

**Review by ikhoon — APPROVED** [review]
Thanks, @minwoox!

**Pull request merged**